src/pages/messages/TasksPage.tsx

```diff
@@ -140,24 +142,24 @@ const TasksPage = () => {
 
   const ticketViewedByQuery = ticketUnreadFilter && user ? `(NOT(viewedBy.username:"${user.nvaUsername}"))` : '';
 
-  const ticketsSearchQuery = urlSearchQuery ?? '';
-
-  const ticketQueryString = [
-    ticketsSearchQuery,
-    ticketTypeQuery,
-    ticketStatusQuery,
-    ticketAssigneeQuery,
-    ticketViewedByQuery,
-  ]
+  const ticketQueryString = [queryParam, ticketTypeQuery, ticketStatusQuery, ticketAssigneeQuery, ticketViewedByQuery]
     .filter(Boolean)
     .join(' AND ');
 
-  const ticketQuery = `${ticketQueryString}&viewingScope=${organizationScope.join(',')}${excludeSubunitsQuery}`;
+  const ticketSearchParams: FetchTicketsParams = {
+    query: ticketQueryString,
+    results: rowsPerPage,
+    from: (page - 1) * rowsPerPage,
+    orderBy: searchParams.get(TicketSearchParam.OrderBy) as 'createdDate' | null,
+    sortOrder: searchParams.get(TicketSearchParam.SortOrder) as 'asc' | 'desc' | null,
+    viewingScope: organizationScope.length > 0 ? organizationScope.join(',') : null,
+    excludeSubUnits: excludeSubunits,
+  };
 
   const ticketsQuery = useQuery({
-    enabled: isOnTicketsPage,
-    queryKey: ['tickets', rowsPerPage, page, ticketQuery],
-    queryFn: () => fetchTickets(rowsPerPage, (page - 1) * rowsPerPage, ticketQuery),
+    enabled: isOnTicketsPage && !!institutionUserQuery.data,
```

Comment:
et lite problem at denne alltid vil stå i loading state (`isLoading=true`) om henting av user feiler.. usikker på hvordan løse det på en god og enkel måte. Blir fort mye herk for en liten cornercase føler jeg 🤔

Reply:
doh.. er jo bare å sjekke på `isLoading`